reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/LiteralToUniformReductionOpportunities.java

```diff
@@ -36,7 +38,21 @@ private LiteralToUniformReductionOpportunities() {
 
     for (TranslationUnit tu : shaderJob.getShaders()) {
       new StandardVisitor() {
-        // TODO: something.
+
+        @Override
+        public void visitIntConstantExpr(IntConstantExpr intConstantExpr) {
```

Comment:
Is support for UintConstantExpr coming in the next PR?  (If so that's fine, but let's not forget.)

Reply:
Support included in the new PR.